lib/src/main/java/fi/epicbot/toster/executor/android/AndroidExecutor.kt

```diff
@@ -105,7 +105,11 @@ internal open class AndroidExecutor(
             // TODO make measurement only once
             val rawData = "wm size".adbShell()
             val rawSize = rawData.replace("Physical size: ", "").split("x")
-            swipeMove.toMove(width = rawSize[0].toInt(), height = rawSize[1].toInt())
+            swipeMove.toMove(
+                swipeOffset = config.swipeOffset,
+                width = rawSize[0].toInt(),
+                height = rawSize[1].toInt()
```

Comment:
Should we check there index? Is it possible to have index exception?

Reply:
not really, it's standard output for adb command. If it changed, I guess several command can broke down